udata_hydra/crawl/check_resources.py

```diff
@@ -58,6 +58,7 @@ async def check_resource(
     url: str,
     resource_id: str,
     session,
+    force_analysis: bool = False,
```

Comment:
Could it be close to `worker_priority` in the params order since it's kind of the same vibe?